wasm/silkworm_wasm_api.cpp

```diff
@@ -44,14 +44,7 @@ uint8_t* bytes_data(Bytes* str) { return str->data(); }
 
 size_t bytes_length(const Bytes* str) { return str->length(); }
 
-intx::uint256* new_uint256_le(uint64_t a, uint64_t b, uint64_t c, uint64_t d) {
-    auto out{new intx::uint256};
-    out->lo.lo = a;
-    out->lo.hi = b;
-    out->hi.lo = c;
-    out->hi.hi = d;
-    return out;
-}
+intx::uint256* new_uint256_le(uint64_t a, uint64_t b, uint64_t c, uint64_t d) { return new intx::uint256(a, b, c, d); }
```

Comment:
```suggestion
intx::uint256* new_uint256_le(uint64_t a, uint64_t b, uint64_t c, uint64_t d) { return new intx::uint256{a, b, c, d}; }
```
Don't you use `{}` for initialization?